grails-app/taglib/org/pih/warehouse/SelectTagLib.groovy

```diff
@@ -486,6 +486,10 @@ class SelectTagLib {
             ActivityCode activityCode = attrs.activityCode ?: null
             attrs.from = locationService.getAllLocations().sort { it?.name?.toLowerCase() }
 
+            if (attrs.hasOrganization) {
+                attrs.from = attrs.from.findAll { !!it.organization }
```

Comment:
I think the double negation is not needed here, `it.organization` should be enough.